contents/css/_components.schedule-item.css

```diff
@@ -15,8 +15,20 @@
   &__title {
     color: var(--c-black);
     font-size: 1.5rem;
-    margin-bottom: .7rem;
-    margin-top: .7rem;
+    line-height: 3.2rem;
```

Comment:
line-height doesn't need a unit
and will this look ok if it breaks to two lines? (does it ever?)

Reply:
Oh-oh, I knew I forgot about something… Will double check and replace it with an adjustments to margins instead.